packages/elements/src/text-field/index.ts

```diff
@@ -231,11 +230,11 @@ export class TextField extends FormFieldElement {
 
   /**
    * @param error existing state of error
-   * @returns true if there is no error and browser is IE11 and minLength more than 0 and value exists
+   * @returns true if there is no error and minLength more than 0 and value exists
    */
   /* c8 ignore start */
   protected shouldValidateForMinLength (error: boolean): boolean {
-    return !!(!error && isIE && this.minLength && !!this.value);
+    return !!(!error && this.minLength && !!this.value);
```

Comment:
If I understand correctly, I think function `shouldValidateForMinLength`  can be remove.

Reply:
Thanks. I double check and it's removable as `checkValidity()` would already validate against the constraints provided.